hypervisor/vm_states.go

```diff
@@ -153,7 +154,8 @@ func (ctx *VmContext) updateInterface(index int, result chan<- error) {
 
 func (ctx *VmContext) setWindowSize(containerId, execId string, size *WindowSize) {
 	var session uint64
-	if execId != "" {
+
+	if execId != "" && !strings.HasSuffix(execId, "-init") {
```

Comment:
from your change, seems like containerId is not null? can you use `execId != containerId+"-init"` directly here?